### DISCUSSION TITLE
Xrefs disable

### DISCUSSION THREAD

**Discussion by ilpadrinohack** [MatthiasValvekens/pyHanko#371] [Q&A]
Hi

When trying to sign pdf, it gives me the following error:

pyhanko.sign.general.SigningError: Attempting to sign document with hybrid cross-reference sections while hybrid xrefs are disabled

Any ideas??
Thanks in advance

**Comment by MatthiasValvekens** [comment]
This is in the FAQ: https://pyhanko.readthedocs.io/en/latest/faq.html#i-m-getting-an-error-about-hybrid-reference-files-when-trying-to-sign-validate-a-file-what-gives ;)

**Comment by fhgisi** [comment]
The issue occurred for me when using *.pdf files that have been created with Excel 365 (Version 2504 Build 16.0.18730.20220) 64 Bit) .

Using` strict=False` as described in the linked FAQ worked for me. 

```
def _load_pdf(file_path):
    with open(file_path, "rb") as f:
        pdf_bytes = f.read()

    pdf_stream = BytesIO(pdf_bytes)
    pdf_writer = IncrementalPdfFileWriter(pdf_stream, strict=False) 
    return pdf_writer 
```

**Reply by MatthiasValvekens** [reply]
Yes, indeed, MS Office (in)famously generates this kind of file with its default print-to-PDF functionality.
